_examples/custom_engine_tarantool/tntserver/centrifuge.lua

```diff
@@ -270,7 +254,7 @@ function centrifuge.history(channel, since_offset, limit, include_pubs, meta_ttl
     end
     local epoch = tostring(now)
     box.begin()
-    box.space.meta:upsert({channel, 0, epoch, meta_exp}, {{'=', 1, channel}, {'=', 4, meta_exp}})
+    box.space.meta:upsert({channel, 0, epoch, meta_exp}, {{'=', 'channel', channel}, {'=', 'meta_exp', meta_exp}})
```

Comment:
here is a typo – `meta_exp` should be `exp`, I'll fix this right after merging pull request.

Reply:
merged and fixed